packages/grafana-ui/src/components/Logs/LogDetailsRow.tsx

```diff
@@ -126,14 +126,13 @@ class UnThemedLogDetailsRow extends PureComponent<Props, State> {
                       href={link.href}
                       target={'_blank'}
                       onClick={
-                        link.onClick
-                          ? event => {
```

Comment:
This caused TS checks failed with "implicit any" error.

Reply:
you could check if onClick exists this way:
```
onClick={() => {
  if(onClick) {
    ...
```